lib/Drush/Symfony/BootstrapCompilerPass.php

```diff
@@ -0,0 +1,31 @@
+<?php
```

Comment:
Possibly; I am still uncertain about our best strategy vis-a-vis freezing the container.